docker/images/noetic/Dockerfile

```diff
@@ -110,7 +110,7 @@ RUN mkdir -p /home/$USER/.ccache $USER_WS/src
 COPY --from=cacher --chown=$USER:$GROUP /tmp/ws/ $USER_WS/
 
 RUN sudo apt-get update \
-  && rosdep update \
+  && . /opt/ros/noetic/setup.sh && rosdep update \
```

Comment:
nit^ $\infty$: I find it useful to have one command per line in Dockerfiles, at least the ones that require && separation.

Reply:
Done in 913faa00426665617399ad94015c45f730a02699